### PR TITLE
do not subscribe to push if already subscribed

### PR DESCRIPTION
**Pull request by p0o**

### CODE REVIEW

File: src/auth/pushpadHelper.js [hunk 1]
```diff
@@ -8,62 +8,70 @@ export const initPushpad = (uid, username, token) => {
...
-            username,
-            token,
-          }),
-        })
+        const { signatureId } = res;
+
+        pushpad('init', PUSHPAD_PROJECT_ID);
+        pushpad('uid', `${uid}`, signatureId);
+        pushpad('subscribe', (isSubscribed) => {
+          if (!isSubscribed) {
```
Comment (nil1511) [inline src/auth/pushpadHelper.js]: Can this condition be true?

Comment (p0o) [inline src/auth/pushpadHelper.js]: @nil1511 what you mean?

Comment (nil1511) [inline src/auth/pushpadHelper.js]: Signature is fetched only when user is not subscribed( inferred from line 25). So I thought isSubscribed should be always false here.
If isSubscribed is always false how will it go to line 57.

variable should be named differently If there purpose is different.

Comment (p0o) [inline src/auth/pushpadHelper.js]: I understand what you mean but it doesn't work like that. signature is fetched if user is not subscribed so after that pushpad will ask user to accept the permission, if user accept this time, the callback will return true so the code will be executed in line 57.

Comment (nil1511) [inline src/auth/pushpadHelper.js]: Cool. Can you change the variable name to like permissionAccepted or something like that. It is confusing using same variable name.

Comment (p0o) [inline src/auth/pushpadHelper.js]: It's the default name from pushpad API and the variable is the same and not different meaning. The thing is the first time we want to make sure to don't repeat the process for already subscribed users and second time if user subscribed we want to send something to our server.

**Pull request merged**